packages/sanes-chrome-extension/src/routes/tx-request/index.dom.spec.ts

```diff
@@ -70,5 +69,5 @@ describe('DOM > Feature > Transaction Request', (): void => {
     await sleep(2000);
     //TODO: Check here that share request rejection has been reject successfuly and permanent
```

Comment:
This comment should be removed because the whitelist sender is well tested in another place.